apps/dalmatiner_db/src/dalmatiner_tcp.erl

```diff
@@ -45,10 +45,6 @@ loop(Socket, Transport, State) ->
                     {ok, Bs} = metric:list(),
                     Transport:send(Socket, dproto_tcp:encode_metrics(Bs)),
```

Comment:
Remove the dynamic function call on line 46. Only modules that define callbacks should make dynamic calls.